two-factor.php

```diff
@@ -142,9 +142,13 @@ function wpcom_vip_enforce_two_factor_plugin() {
 		// Calcuate two factor authentication support outside map_meta_cap to avoid callback loop
 		// see: https://github.com/Automattic/vip-go-mu-plugins/pull/1445#issuecomment-592124810
 		$is_user_using_two_factor = Two_Factor_Core::is_user_using_two_factor();
-		add_filter( 'wpcom_vip_is_user_using_two_factor', function() use ( $is_user_using_two_factor ) {
-			return $is_user_using_two_factor;
-		}, 10 );
+		add_filter( 
+			'wpcom_vip_is_user_using_two_factor',
+			function() use ( $is_user_using_two_factor ) {
+				return $is_user_using_two_factor;
+			}, 
+			10 
```

Comment:
@cojennin Since `10` is the default, let's just leave out that argument.

Reply:
Sure, all set